dbt_subprojects/dex/macros/models/_project/in-flight-decoding/uniswap_v2/uniswap_v2_trades.sql

```diff
@@ -0,0 +1,77 @@
+{% macro uniswap_v2_forks_trades(
+    blockchain = null
+    , dex_type = 'uni-v2'
+    , project = null
+    , version = null
+    , Pair_evt_Swap = null
+    , Factory_evt_PairCreated = null
+    , pair_column_name = 'pair'
+    )
+%}
+
+WITH evt_swap AS (
+    SELECT block_number
+        , block_time
+        , to
+        , contract_address
+        , tx_hash
+        , index
+        , amount0In
+        , amount0Out
+        , amount1In
+        , amount1Out
+    FROM {{ Pair_evt_Swap }}
+    {% if is_incremental() %}
+    WHERE {{ incremental_predicate('block_time') }}
+    {% endif %}
```

Comment:
I've never had to deal with this incremental stuff much before, might moving this down to the end of the macro resolve the merge errors?